### DISCUSSION TITLE
Is it possible to render html from a bound variable?

### DISCUSSION THREAD

**Discussion by ghost** [mikke89/RmlUi#291] [Q&A] [answered]
I'm trying to do something like this:

```
<rml>
    <head>
        <link type="text/rcss" href="rml.rcss"/>
        <link type="text/rcss" href="default.rcss"/>
        <link type="text/rcss" href="ChatWindow.rcss"/>
    </head>
    <body>
        <div class="chat" data-model="ChatWindow">
          <div data-for="log : logs">
            {{ log }}<br />
          </div>
        </div>
    </body>
</rml>
```

where logs is ["Hello world!", "This is <div style=\"color: red;\">red</div>!"]. I'd like the text contents to be rendered as html instead of just a string. I was able to do something similar in the Flask/Jinja python libraries using the syntax {{ log | safe }} and it seemed really convenient.

**Comment by mikke89** [comment, marked as answer]
Hi.

You can use the [`data-rml`](https://mikke89.github.io/RmlUiDoc/pages/data_bindings/views_and_controllers.html#data-rml) view for this purpose. I think the following should work for you:

```html
<div data-for="log : logs" data-rml="log + '<br/>'"/>
```

Let me know how it goes :)
